decidim-core/app/views/decidim/open_data/index.html.erb

```diff
@@ -41,6 +41,20 @@
       <% end %>
     </div>
 
+    <div class="editor-content">
+      <ul>
```

Comment:
I would add some kind of header, like "Component data" , like it is generated in the Readme.